WooCommerce/Classes/ViewRelated/CardPresentPayments/ReceiptViewController.swift

```diff
@@ -35,25 +48,48 @@ final class ReceiptViewController: UIViewController {
 private extension ReceiptViewController {
     func configureToolbar() {
         title = Localization.title
-        navigationItem.rightBarButtonItem = UIBarButtonItem(image: .print,
-                                                            style: .plain,
-                                                            target: self,
-                                                            action: #selector(printReceipt))
+        navigationItem.rightBarButtonItems = [
+            MFMailComposeViewController.canSendMail() ?
```

Comment:
Same as before, feels more natural that the logic to ascertain whether it can send email or not stays in the view model, but again, I can see that it is easier to do it this way 😃

Reply:
Yea good idea to move this to the view model, I made this move in https://github.com/woocommerce/woocommerce-ios/pull/6833/commits/7474efaff66fb1eca59868603a20555f470311b7 - though it's hard to unit test this 😐